### PR TITLE
Some new professions

### PR DESCRIPTION
**Pull request by LariAnjelo**
#### Summary
Content "New professions."

#### Purpose of change

I just wanted a Historian profession to exist in game. Then, I wanted even more professions. This PR will be focused on that: The addition of even more professions to Cataclysm:Dark Days Ahead.

#### Describe the solution

Adding the Historian profession. For the skills, those are as following:

Archery: 1, because of their basic knowledge of historical weapons;
Computers: 1, because of their experience with digital archives and databases;
Speaking: 2, since a historian has some experience giving lectures and presentations;
Survival: 1, because of their nowledge of historical survival techniques and foraging from research on past societies;

I'll do some research to discover if any proficiency would be good too. Thank you all for the attention. There will be also the addition of a notebook item and some history books, which are only for fluff purposes, or just for fun, eh.

And now, some other professions. These will be as follows:

- Librarian
The clothing a traditional librarian would wear. The library card is only a symbolic item. Plus, they start with novels and a manual.

- Competitive eater
The full, professional version of the competitive eater background. Starts with a hamburger themed T-shirt and food.

- Writer
A career writer. Starts with a novel.

- Stocker
A supermarket stocker. No big skills, but starts with an cardboard box with some stuff

- Cashier
A supermarket cashier. Starts with some spare change.

- Store manager
The store manager (now we have an almost full set of supermarket professions), the manager badge is a symbolic item.

- Lobsterman
Since the game takes place in New England, I thought the Lobsterman was a ok addition. Starts with sightly different equip than the fisher, plus a lobster cage, some lobster meat and a lobster.

- Veterinarian
A veterinarian profession, kind of a "weaker" medical resident. Starts with a petpack and antiparasitic drug. The veterinarian badge is a symbolic item.

- Serial killer
Kinda the predecessor of the Death Row Convict. Starts with acceptable skills and some basic gear. The photograph, the coin, the mask are symbolic item, they also just killed someone.

#### Describe alternatives you've considered

Adding a History Professor instead but it sounded too specific for me, since we already have a teacher/professor profession.

#### Testing

Should work just fine.

### CODE REVIEW

**Comment by NetSysFire** [comment]
You messed up the PR validation workflow since you renamed the summary section to your description which should be in its own section down below. Revert it back to Summary and use something more descriptive than "Adds a new profession" that is actually useful in the changelog.

**Comment by LariAnjelo** [comment]
> You messed up the PR validation workflow since you renamed the summary section to your description which should be in its own section down below. Revert it back to Summary and use something more descriptive than "Adds a new profession" that is actually useful in the changelog.

I believe I did it right this time. I'm sorry for the messing up of stuff.

**Comment by NetSysFire** [comment]
For reference: I would make the PR summary "New profession: Historian" or something like this. Your personal narration does not go there. It goes in purpose of change.

File: data/json/professions.json [hunk 1]
```diff
@@ -971,6 +971,341 @@
...
+      },
+      "female": {
+        "entries": [
+          { "item": "bra" },
+          { "item": "boy_shorts" }
+        ]
+      }
+    },
+    "age_lower": 17
+  },
```
Comment (kevingranade) [inline data/json/professions.json:1115]: This is incredibly silly/niche. pass.
```suggestion
```

File: data/json/professions.json [hunk 2]
```diff
@@ -971,6 +971,341 @@
...
+      },
+      "female": {
+        "entries": [
+          { "item": "bra" },
+          { "item": "panties" }
+        ]
+      }
+    },
+    "age_lower": 18
+  },
```
Comment (kevingranade) [inline data/json/professions.json:1153]: Professions need to be somewhat mechanically distinct, this isn't.
```suggestion
```

File: data/json/professions.json [hunk 3]
```diff
@@ -971,6 +971,341 @@
...
+      "female": {
+        "entries": [
+          { "item": "hairpin" },
+          { "item": "bra" },
+          { "item": "panties" }
+        ]
+      }
+    },
+    "age_lower": 21
+  },
```
Comment (kevingranade) [inline data/json/professions.json:1308]: These all have the same problem, professions are "roleplay hooks", but they need to be mechanically distinct in order to support that, we don't need dozens of different practically identical professions only separated by their descriptions.
```suggestion
```

File: data/json/professions.json [hunk 4]
```diff
@@ -4948,6 +5321,43 @@
...
+          { "item": "fish_bait", "charges": 15 },
+          { "item": "lighter", "charges": 100 },
+          { "item": "knife_hunting", "container-item": "sheath" },
+          { "item": "net", "custom-flags": [ "auto_wield" ] }
+        ]
+      },
+      "male": { "entries": [ { "item": "boxer_shorts" } ] },
+      "female": { "entries": [ { "item": "bra" }, { "item": "panties" } ] }
+    }
+  },
```
Comment (kevingranade) [inline data/json/professions.json:5360]: This is just silly, fishermen aren't walking around hauling lobster pots and chunks of smoked meat.
The survival number is also bizzarely high?
```suggestion
```

File: data/json/professions.json [hunk 5]
```diff
@@ -1989,6 +2324,45 @@
       "female": { "entries": [ { "item": "bra" }, { "item": "panties" } ] }
     }
   },
+  {
```
Comment (kevingranade) [inline data/json/professions.json:2327]: This one has potential, though I haven't cross-checked against the full list of professions.

File: data/json/professions.json [hunk 6]
```diff
@@ -971,6 +971,341 @@
...
+        "entries": [
+          { "item": "hairpin" },
+          { "item": "bra" },
+          { "item": "panties" },
+          { "item": "skirt", "variant": "skirt_black" }
+        ]
+      }
+    },
+    "age_lower": 20
+  },
```
[the remark this answers is not in the document]
Comment (LariAnjelo) [reply data/json/professions.json:1068]: I've spent the last few days and nights working on that, this makes me feel really demotivated. Nah just kidding, you suck. Imma make this a mod eh.

**Comment by ervin210** [comment]
You all stolen my coin and  all my repository I had. Hacking me.   And all take money and me. Not.    Me get only problem
S.

**Pull request closed without merging**